src/example-2.go

```diff
@@ -4,14 +4,40 @@ import (
 	"context"
 	"fmt"
 	"github.com/zeebe-io/zeebe/clients/go/pkg/zbc"
+	"os"
 )
 
-const BrokerAddr = "0.0.0.0:26500"
+const ZeebeAddr = "0.0.0.0:26500"
 
+/*
+Sample application that connects to a cluster on Camunda Cloud, or a locally deployed cluster.
+
+When connecting to a cluster in Camunda Cloud, this application assumes that the following
+environment variables are set:
+
+ZEEBE_ADDRESS
+ZEEBE_CLIENT_ID
+ZEEBE_CLIENT_SECRET
+ZEEBE_AUTHORIZATION_SERVER_URL
+
+Hint: When you create client credentials in Camunda Cloud you have the option
+to download a file with above lines filled out for you.
+
+When connecting to a local cluster or node, this application assumes default port and no
+authentication or encryption enabled.
+*/
 func main() {
+  gatewayAddr := os.Getenv("ZEEBE_ADDRESS")
+  plainText:= false
+
+  if (gatewayAddr == "") {
+    gatewayAddr = ZeebeAddr
+    plainText = true
+  }
```

Comment:
The formatting is off and there shouldn't be parentheses around the condition in the `if`.
```suggestion
    gatewayAddr := os.Getenv("ZEEBE_ADDRESS")
    var plainText bool

    if gatewayAddr == "" {
        gatewayAddr = ZeebeAddr
        plainText = true
    }
```